src/applications/personalization/profile360/vet360/util/local-vet360.js

```diff
@@ -5,7 +5,8 @@ export function isVet360Configured() {
   return [
     'staging.vets.gov',
     'www.vets.gov',
-    'preview.va.gov'
+    'preview.va.gov',
+    'staging.va.gov'
```

Comment:
@ncksllvn @BBarone1 - Can you think of any other sites we should add to this list at this point?